packages/core/src/app/payment/paymentMethod/PaymentMethodTitle.test.tsx

```diff
@@ -1,5 +1,6 @@
 import { CheckoutService, createCheckoutService, StoreConfig } from '@bigcommerce/checkout-sdk';
-import { mount } from 'enzyme';
+import { cleanup, render, screen } from '@testing-library/react'
+import '@testing-library/jest-dom'
```

Comment:
Is there any special reason for `import @testing-library/jest-dom` again?
As you can see  on the first line of `jest-setup.ts`, it is already there.

Reply:
yes, we can delete it.

Reply:
without it I'm getting an error `Property 'toHaveAttribute' does not exist on type 'JestMatchers<HTMLElement>'.`